### PR TITLE
 JS function to prevent the zombie apocalypse

### PR DESCRIPTION
**Pull request by nilkun**

### CODE REVIEW

File: meetup-8/solutions/4-grenades-and-zombies[JAVASCRIPT].js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    // VARIABLES
+    const dimensions = board.length;
+    let zombiesFound = 0;
+    let startX = 0;
+
+    // POPULATE GRID WITH ZEROES
+    let solutionGrid = Array(dimensions).fill(0).map(x => Array(dimensions).fill(0));
+    
+    // I like to start from y
+    for(let y = 0; y < dimensions; y++)
```
Comment (yanarchy) [inline meetup-8/solutions/4-grenades-and-zombies[JAVASCRIPT].js]: JS is a bit different from C and other languages---the curly brackets hug the preceding closing parens.

Can you change these to: `){`?

**Comment by nilkun** [comment]
Thank you for the feedback!

**Comment by yanarchy** [comment]
Woohoo! Not sure if you want more feedback! If you're good, then I'll approve and merge 👍

**Comment by nilkun** [comment]
Wonderful! I'm good with the code, but I'd really appreciate feedback about implementation!

**Comment by yanarchy** [comment]
@nilkun Bah, sorry! I have been super busy and haven't had a chance to go through your code :(

**Comment by yanarchy** [comment]
@nilkun Took a quickie look at it---it's a great start. I noticed that there are a lot of redundancies---if you organize it a bit and abstract out some of the different things you're doing, you will probably find them and stay DRY. Also, I think because you do Y first, it returns [Y, X] instead of [X, Y]. Again, take this feedback with a grain of salt because I didn't have a chance to go over it carefully. I'm going to merge it so people can see it.

Come to our [next algorithms meetup](https://www.meetup.com/CodeChrysalis/events/clvfbqyxlblc/)! It's at Zehitomo this time.

**Pull request merged**